setup.cfg

```
@@ -60,9 +61,10 @@ envlist =
     # tests against released versions
     py{38,39}-dj{42}-redislatest
     py{310,311,312}-dj{42,50,51}-redislatest
+    py313-dj51-redislatest
```

why not do like on line 67?

---

This line aims to add Python 3.13 (supported only by Django 5.1) and the latest Redis to the matrix. Not sure how it can be more like line 67, an example would be nice.

---

```suggestion
    py313-dj51-redis{latest,master}
```
and remove `py313-dj51-redismaster` from line 66

---

Oh, I see. I can apply that, but there is contextual grouping between `tests against released versions` and `tests against unreleased versions` in that block. Do we want to change that?

---

`python3.13` is not yet supported by `redis-py` officially so I guess we can consider it unreleased 😄

---

`redis-py` started supporting Python 3.13 in their 6.0.0. release: https://github.com/redis/redis-py/releases/tag/v6.0.0